homeassistant/components/plugwise/climate.py

```diff
@@ -156,6 +157,9 @@ def extra_state_attributes(self) -> Mapping[str, Any] | None:
     @plugwise_command
     async def async_set_temperature(self, **kwargs: Any) -> None:
         """Set new target temperature."""
+        if ATTR_HVAC_MODE in kwargs:
```

Comment:
I think these new lines should go below line 175 so we check that the temperature is fine first.

Reply:
Good suggestion! Implemented.